.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,26 @@
+<!-- Remember to use a meaningful title that references the story ID (or story IDs) that this PR fixes. For example: [AB-123] Adds some cool new feature -->
+
+* **What kind of change does this PR introduce?**
```

Comment:
Did you want to append the pull request description to the changelog when we release?

If so is this the format that you want?  Or did you want to make use of titles instead of questions for each section so that it makes more sense for the person reading the changelog...

See #24 for example